packages/ui-markdown-editor/src/index.js

```diff
@@ -130,10 +129,15 @@ export const MarkdownEditor = (props) => {
     const CICERO_MARK_DOM = slateTransformer.toCiceroMark(SLATE_DOM);
     const HTML_DOM = htmlTransformer.toHtml(CICERO_MARK_DOM);
     const MARKDOWN_TEXT = ciceroMarkTransformer.toMarkdown(CICERO_MARK_DOM);
+    const [imageNode] = Editor.nodes(editor, { match: n => n.type === 'image' });
```

Comment:
@irmerk @DianaLease @dselman Is this the best way to detect if the operation applies to an 'image'?

Reply:
@irmerk @DianaLease @dselman Also tell me the best way to detect if the operation applies to  'text'

Reply:
We have precedent here for detecting an image, see [L175](https://github.com/accordproject/web-components/pull/298/files#diff-46ada2eb1165549f5e15ed60cb7838af16bb6ab14c15530502dde1c2590c2e2cR175) under `handleDrop` in this file. 

I'd look for other inspiration on a text node.

Reply:
@irmerk Sure.